client/common/cmdline.c

```diff
@@ -1704,15 +1696,44 @@ static BOOL setSmartcardEmulation(const char* value, rdpSettings* settings)
 	return TRUE;
 }
 
+BOOL option_starts_with(const char* what, const char* val)
+{
+	WINPR_ASSERT(what);
+	WINPR_ASSERT(val);
+	const size_t wlen = strlen(what);
+
+	return _strnicmp(what, val, wlen) == 0;
+}
+
+BOOL option_ends_with(const char* str, const char* ext)
+{
+	WINPR_ASSERT(str);
+	WINPR_ASSERT(ext);
+	const size_t strLen = strlen(str);
+	const size_t extLen = strlen(ext);
+
+	if (strLen < extLen)
+		return FALSE;
+
+	return _strnicmp(&str[strLen - extLen], ext, extLen) == 0;
+}
+
+BOOL option_equals(const char* what, const char* val)
+{
+	WINPR_ASSERT(what);
+	WINPR_ASSERT(val);
+	return _stricmp(what, val) == 0;
+}
+
 static int parse_on_off_option(const char* value)
 {
 	WINPR_ASSERT(value);
 	const char* sep = strchr(value, ':');
 	if (!sep)
 		return 1;
-	if (_strnicmp("on", &sep[1], 3) == 0)
+	if (_stricmp("on", &sep[1]) == 0)
```

Comment:
Since you already replaced such checks in other places directly with `option_equals` too, this could be done here too (same with the "off` option a few lines below.

Reply:
done